src/drivers/amd/common/acp_dma.c

```diff
@@ -263,7 +271,11 @@ static int acp_dma_status(struct dma_chan_data *channel,
 {
 	status->state = channel->status;
 	status->flags = 0;
+#ifdef __ZEPHYR__
+	status->timestamp = sof_cycle_get_64();
+#else
 	status->timestamp = timer_get_system(timer_get());
+#endif
```

Comment:
Fwiw, on Intel side we only had to keep this sort of change temporary as we integrated our DMA/DAI drivers into Zephyr, once we had the drivers in Zephyr we were able to deprecate the old driver and remove them.

Reply:
For non-Zephyr cases, the function definition of sof_cycle_get_64() does not exist.